web/src/echo.ts

```diff
@@ -367,14 +460,23 @@ export function update_message_lists({old_id, new_id}) {
     }
 }
 
-export function process_from_server(messages) {
+export function process_from_server(messages: ServerMessage[]): ServerMessage[] {
     const msgs_to_rerender = [];
     const non_echo_messages = [];
```

Comment:
These two arrays are implicitly of type `any[]` we should type them with `Message[]`.

Reply:
Done.